src/OXRS_LCD.cpp

```diff
@@ -733,3 +759,97 @@ void OXRS_LCD::_set_backlight(int val)
 {
   ledcWrite(BL_PWM_CHANNEL, 255*val/100); 
 }
+
+
+
+
+// Bodmers BMP image rendering function
+
+void OXRS_LCD::_drawBmp(const char *filename, int16_t x, int16_t y) 
```

Comment:
If we don't have SPIFFs can we have a simple OXRS logo built-in - the one you have in the current version - as a fallback?